### PR TITLE
Curate UI Feedback

### PR DESCRIPTION
**Pull request by tarwin**
feat: renamed `Curate` button to `ActionButton` to make clearer
feat: added `CurateButton`
feat: `CurateButton` added loading indicator on action

Sorry this is my 3rd attempt at doing this the right way. Hopefully everything is in order now, and apologies for the previous attempts.

### CODE REVIEW

File: src/components/button/index.js [hunk 1]
```diff
@@ -90,3 +92,30 @@ export const Burn = ({ children = null, selected }) => {
...
 }
+
+export const CurateButton = (props) => {
+  const [waiting, setWaiting] = useState(false)
+  const context = useContext(HicetnuncContext)
+
+  const tryCurate = async() => {
+    setWaiting(true)
+    context.curate(props.tokenId)
+    .finally(() => {
```
Comment (tonypee) [inline src/components/button/index.js:103]: tabbing

**Comment by andrevenancio** [comment]
There's some conflicts. can you update this PR against the `develop` branch too?

**Comment by tarwin** [comment]
I'm a little confused. Should we be developing against the `main` or `develop` branch?

**Comment by tarwin** [comment]
I'll add this again against the `develop` branch ASAP.

**Pull request closed without merging**